LiteCore/Query/SQLiteN1QLFunctions.cc

```diff
@@ -639,16 +661,30 @@ namespace litecore {
 
     // length() returns the length in characters of a string.
     static void length(sqlite3_context* ctx, int argc, sqlite3_value **argv) noexcept {
+        if (sqlite3_value* mnArg = passMissingOrNull(1, argv); mnArg != nullptr) {
```

Comment:
Passing `argc` instead of `1` seems safer, since it won't break if we change the number of arguments of this function, or if copying/pasting code into a new function definition.

(This goes for many of the diffs below too)

Reply:
Accepted: going to find all occurrences and change them to argc